pairing/bls12381/bls12381_test.go

```diff
@@ -730,31 +733,47 @@ func BLSBenchmark(b *testing.B, curveOption string) {
 		}
 	}
 
+	// Prepare masks for aggregation
+	maskG1, err := sign.NewMask(pubKeysOnG1, pubKeysOnG1[0])
+	if err != nil {
+		panic(err)
+	}
+	maskG2, err := sign.NewMask(pubKeysOnG2, pubKeysOnG2[0])
+	if err != nil {
+		panic(err)
+	}
 	for _, n := range numSigs {
+
 		// Benchmark aggregation of public keys
 		b.Run(fmt.Sprintf("AggregatePublicKeys-G1 on %d signs", n), func(bb *testing.B) {
 			for j := 0; j < bb.N; j++ {
-				result = schemeOnG1.AggregatePublicKeys(pubKeysOnG1[:n]...)
+				result, err = schemeOnG1.AggregatePublicKeys(maskG1)
+				if err != nil {
+					panic(err)
```

Comment:
why panic instead of a `require.NoError`?

Reply:
No good reason apparently, I didn't expect `*time.T` and `*time.B` to be swappable, but it seems to work that way :+1: